src/main/webapp/assets/modules/talent-search/main/main.ser.js

```diff
@@ -46,9 +47,13 @@ techlooper.factory("tsMainService", function (jsonValue, $http, $location) {
       });
     },
     searchTalent: function () {
-      var request = {
-        keywords: searchRequest.keywords.getValue().split(","),
-        locations: searchRequest.locations.getValue().split(",")
+      //var request = {
+      //  keywords: searchRequest.keywords.getValue().split(","),
+      //  locations: searchRequest.locations.getValue().split(",")
+      //}
+      var request = {};
+      for (var prop in searchRequest) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.